packages/p2p/README.md

```diff
@@ -0,0 +1,59 @@
+<h1 align="center">
+  deriv-p2p
+</h1>
+
+## Requirements
+
+-   node
+-   npm
+
+## Editor helpers
+
+-   Prettier setup in your editor https://prettier.io/
+-   Stylelint setup in your editor https://stylelint.io/
+-   Eslint setup in your editor https://eslint.org/
+
+## 🚀 Quick start
+
+1.  **Install your dependencies:**
+
+    ```sh
+    npm install
```

Comment:
Would it be preferred to use `npm ci` instead?

Reply:
true, since package.lock is part of git